### DISCUSSION TITLE
Функция копирования в буфер обмена

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#264] [Есть идея, а что если...]
Иногда возникает необходимость скопировать значения из одного или нескольких полей или результатов поиска в буфер обмена. Сейчас это возможно только через выделение в таблице или кнопкой в контекстной панели. 

Было бы здорово иметь функцию, аналогичную селектам (row, list, rowlist), которая бы копировала значения полей (или их отображение) из выборки сразу в буфер.

**Comment by ttm-alexeypolunin** [comment]
Хорошая мысль. Запишемс, может даж быстро сделаем тк там база то есть под нее.

**Reply by ttm-alexeypolunin** [reply]
В PRO что-то типа proCopyToBuffer — enjoy!

**Reply by erpic-ru** [reply]
proLinkToBuffer
